Wikipedia/View Controllers/RecentSearches/RecentSearchesViewController.m

```diff
@@ -103,8 +96,8 @@ - (void)saveTerm:(NSString*)term
          @"type": @(searchType)
      } atIndex:0];
 
-    if (self.tableDataArray.count > LIMIT) {
-        self.tableDataArray = [self.tableDataArray subarrayWithRange:NSMakeRange(0, LIMIT)].mutableCopy;
+    if (self.tableDataArray.count > recentSearchLimit) {
+        self.tableDataArray = [self.tableDataArray subarrayWithRange:NSMakeRange(0, recentSearchLimit)].mutableCopy;
```

Comment:
use `NSArray wmf_safeSubarrayWithRange:` to avoid an `NSRangeException` when out of bounds.